pytest_coverage

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+pytest --cov-report=term-missing:skip-covered --cov=numjuggler $*
```

Comment:
This script generated an error:

Under Python2:

<pre>(base) <font color="#AF0000">anton@local-Laptop1806 numjuggler&gt; </font>./pytest_coverage 
usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov-report=term-missing:skip-covered --cov=numjuggler
  inifile: /home/anton/work/gh/numjuggler/pytest.ini
  rootdir: /home/anton/work/gh/numjuggler
(base) <font color="#AF0000">anton@local-Laptop1806 numjuggler&gt; </font>pytest --version
This is pytest version 4.0.2, imported from /home/anton/anaconda2/lib/python2.7/site-packages/pytest.pyc
(base) <font color="#AF0000">anton@local-Laptop1806 numjuggler&gt; </font>
</pre>

Under Python3:

<pre>(base) <font color="#AF0000">anton@local-Laptop1806 numjuggler&gt; </font>./pytest_coverage 
usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov-report=term-missing:skip-covered --cov=numjuggler
  inifile: /home/anton/work/gh/numjuggler/pytest.ini
  rootdir: /home/anton/work/gh/numjuggler
(base) <font color="#AF0000">anton@local-Laptop1806 numjuggler&gt; </font>pytest --version
This is pytest version 4.0.2, imported from /home/anton/anaconda3/lib/python3.7/site-packages/pytest.py
setuptools registered plugins:
  pytest-remotedata-0.3.1 at /home/anton/anaconda3/lib/python3.7/site-packages/pytest_remotedata/plugin.py
  pytest-openfiles-0.3.1 at /home/anton/anaconda3/lib/python3.7/site-packages/pytest_openfiles/plugin.py
  pytest-doctestplus-0.2.0 at /home/anton/anaconda3/lib/python3.7/site-packages/pytest_doctestplus/plugin.py
  pytest-arraydiff-0.3 at /home/anton/anaconda3/lib/python3.7/site-packages/pytest_arraydiff/plugin.py
(base) <font color="#AF0000">anton@local-Laptop1806 numjuggler&gt; </font>
</pre>

Reply:
You need pytest-cov module installed. The module is listed in setup.py task_require parameter (in my version).

Reply:
On Python2 or Python3 see [this](https://pythonclock.org/) and [this](https://www.anaconda.com/end-of-life-eol-for-python-2-7-is-coming-are-you-ready/)